pyproject.toml

```
@@ -28,6 +28,7 @@ requires-python = ">=3.11"
 dependencies = [
   "arches>=8.0.4,<8.1.0",
   "python-docx==1.1.0",
+  "arches_controlled_lists @ git+https://github.com/archesproject/arches-controlled-lists.git@main",
```

```suggestion
  "arches_controlled_lists @ git+https://github.com/archesproject/arches-controlled-lists.git@dev/1.0.x",
```
We removed the main branch in favor of development branches